sfge/src/test/java/com/salesforce/graph/symbols/apex/schema/FieldSetTest.java

```diff
@@ -25,19 +28,19 @@ public void setup() {
     }
 
     @ValueSource(
-            strings = {"fs.getDescription()", "fs.getLabel()", "fs.getName()", "fs.getNamespace()"})
+        strings = {"fs.getDescription()", "fs.getLabel()", "fs.getName()", "fs.getNamespace()"})
```

Comment:
Were these formatting changes performed by your IDE or by `yarn build`? Just want to make sure we're not introducing any unintentional changes.

Reply:
Did a yarn build cleanup. Some of these should get standardized. I haven't been controlling indentations.